packages/react-components/src/organisms/__tests__/TeamsTabbedCard.test.tsx

```diff
@@ -6,18 +6,41 @@ import TeamsTabbedCard from '../TeamsTabbedCard';
 const props: ComponentProps<typeof TeamsTabbedCard> = {
   teams: [],
   title: '',
+  disableActiveTab: false,
 };
-it('renders the teams tabbed card', () => {
-  render(
-    <TeamsTabbedCard
-      {...props}
-      teams={[{ displayName: 'team 1', id: '123' }]}
-      title="Example"
-    />,
-  );
-  expect(screen.getByRole('heading').textContent).toEqual('Example');
-  expect(screen.getByTitle('Team')).toBeInTheDocument();
-  expect(screen.getByText(/team 1/)).toBeVisible();
+
+describe('renders the teams tabbed card', () => {
+  const renderTabbedCard = (disableActiveTab: boolean) =>
+    render(
+      <TeamsTabbedCard
+        {...props}
+        teams={[
+          { displayName: 'team 1', id: '123' },
+          { displayName: 'team 2', id: '123', inactiveSince: '2021-01-01' },
+        ]}
+        title="Example"
+        disableActiveTab={disableActiveTab}
+      />,
+    );
+
+  it('renders for an Active Group', () => {
```

Comment:
Should this cover disabled rules for tabs? Check first with the above regarding how we should handle the edge cases to do with disabled tabs.